src/client/common/variables/environmentVariablesProvider.ts

```diff
@@ -96,6 +96,12 @@ export class EnvironmentVariablesProvider implements IEnvironmentVariablesProvid
         this.envVarsService.mergeVariables(process.env, mergedVars); // Copy current proc vars into new obj.
         this.envVarsService.mergeVariables(customEnvVars!, mergedVars); // Copy custom vars over into obj.
         const pathVariable = this.platformService.pathVariableName;
+        if (process.env[pathVariable]) {
```

Comment:
Anything that uses this now has a possible PATH or PYTHONPATH value from the process.env that it would not have had before. Seems like a bit of a big change, but I guess we'll just have to rely on tests catching any fallout on this.

Reply:
> Anything that uses this now has a possible PATH or PYTHONPATH value from the process.env that it would not have had before. Seems like a bit of a big change, but I guess we'll just have to rely on tests catching any fallout on this.

It should be there if it wasn't.